test/built-ins/GeneratorPrototype/throw/try-catch-following-catch.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2013 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 25.3.1.4
+description: >
+    When a generator is puased after a `try..catch` statement, `throw` should
+    interrupt control flow as if a `throw` statement had appeared at that
+    location in the function body.
+---*/
+
+function* g() {
+  yield 1;
+  try {
+    yield 2;
+  } catch (e) {
+    yield e;
+  }
+  yield 3;
+}
+var iter, result, exception;
+
+exception = new Test262Error();
```

Comment:
My concern is that there might be some false positives (or negatives) when using these but maybe I'm worrying too much?